webapp/app/js/perf.js

```diff
@@ -179,6 +179,74 @@ perf.controller('PerfCtrl', [ '$state', '$stateParams', '$scope', '$rootScope',
       });
     }
 
+    function plotOverviewGraph() {
+      $scope.seriesList.forEach(function(series) {
```

Comment:
Could you put a comment here saying something to the effect of "We want to show lines for series in the overview plot, if they are visible"?